src/button/index.js

```diff
@@ -1 +1,2 @@
-export default from './Button';
+export default from './StatefulButton';
+export PureButton from './Button';
```

Comment:
sure! feel free to edit!

On Thu, Apr 21, 2016 at 5:03 PM Francesco <notifications@github.com> wrote:

> In src/button/index.js
> <https://github.com/buildo/react-components/pull/403#discussion_r60596647>
> :
>
> > @@ -1 +1,2 @@
> > -export default from './Button';
> > +export default from './StatefulButton';
> > +export PureButton from './Button';
>
> if StatefulButton is the default maybe we should rename Button to
> PureButton and StatefulButton to Button:
> I find confusing that import Button from '../button'; will import
> StatefulButton instead of Button
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/buildo/react-components/pull/403/files/1ce1b45df056332b9c720b3f428ba9fdb9273aee#r60596647>
>